lib/orbit/rail_line.ex

```diff
@@ -31,4 +31,7 @@ defmodule Orbit.RailLine do
       _ -> :none
     end
   end
+
+  @spec from_route_id(Realtime.Data.route_id()) :: t()
```

Comment:
At a later date we may want to remove `Realtime.Data.route_id()`, or more clearly delineate it from `RailLine`.